study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/EmailBuilder.java

```diff
@@ -192,7 +192,7 @@ private String readRawHtmlContent(File file) {
     }
 
     private String renderHtmlContent(String name, String rawHtml) {
-        var ctx = new VelocityContext(varsCfg.root().unwrapped());
+        var ctx = new VelocityContext(varsCfg.resolve().root().unwrapped());
```

Comment:
Found this bug while working on testboston's config. `resolve()` needs to be called to fully render included files.